sites/all/themes/orwell/sass/component/_main-menu.scss

```diff
@@ -57,12 +57,25 @@ $menu-height__medium: 5rem;
     }
 
     ul {
-      display: flex;
-      justify-content: space-between;
-
+      @include grid-media($xxsmall){
+        display: flex;
+        justify-content: space-between;
+        flex-direction: column;
+        a {
+          &:hover {
+            background: darken($color__geist--red, 10%);
+          }
+        }
+      }
       @include grid-media($medium) {
         margin: 0 $padding__default;
-        display: inline-block;
+        display: inline-block !important; // This is set to !important to prevent menu hidden when shifting between breakpoints without reload. It is nessery since .slideToggle is adding display none, when toggleling menu on mobile.
```

Comment:
Would it be possible to make this as an multi-line comment above the line.

Will make it alot easier to read.